### ISSUE TITLE
CCA icon alternative text in English in the French interface "About" (WCAG SC 11.1.1.1)

### ISSUE THREAD

**Issue by laurencechavet** [ThePacielloGroup/CCAe#325]
**Describe the bug**
In the "About" window (à propos in French), the CCA icon alternative text in English in the French interface

**Desktop (please complete the following information):**
 - OS: Windows 10 64 bits
 - Version CCA 3.4.2

**To Reproduce**
Open the CCA in French, go in the menu "Analyseur de contraste des couleurs", choose "à propos de CCA". Listen with a text reader or use the Developer tools (Ctrl Shift I), the alternative text for the icon should be in French but is in English

**Expected behavior**
The alternative text for the icon is in French and read in French with an text reader

**Screenshots**
![bug_11 1 1 1_alttext4icon](https://github.com/ThePacielloGroup/CCAe/assets/65306416/9ffbf995-88bd-4040-a022-1e3cecd30e33)

**Additional context**
N/A
